package.json

```diff
@@ -88,6 +87,8 @@
     "prettier": "2.0.5",
     "prismjs": "1.21.0",
     "puppeteer": "1.19.0",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
```

Comment:
Why is react suddenly needed?

Reply:
It was included implicitly by Docz before, and the `react-jsonschema-form` library in the form component needs it. I think it was only set as a peer dependency from `react-jsonschema-form`